lib/monopay-ruby/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module MonopayRuby
-  VERSION = "0.1.1"
+  VERSION = "1.1.1"
```

Comment:
the older version resets the younger ones

Reply:
fixed